### PR TITLE
[Zaraz] Post-migration fixes

### PR DESCRIPTION
**Pull request by marciocloudflare**
Removed extra spaces

### CODE REVIEW

File: content/zaraz/advanced/block-zaraz.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ meta:
...
+4. Create a URL pattern that matches the page, domain or subdomain you want to exclude Zaraz from loading on.
+5. Choose _Disable Zaraz_ from the **Then the settings are** dropdown menu.
+6. Click **Save and deploy**.
 
 Refer to [Understanding and configuring Cloudflare Page Rules](https://support.cloudflare.com/hc/articles/218411427) for more information about how Page Rules work.
 
 {{<Aside type="note">}}
 
-If you need to block just one or more events from firing in a tool, we recommend you use [Blocking Triggers](/zaraz/get-started/create-trigger/#blocking-triggers).
+If you need to block just one or more events from firing in a tool, we recommend you use [Blocking Triggers](/zaraz/get-started/create-trigger/#blocking-triggers/).
```
Comment (pedrosousa) [inline content/zaraz/advanced/block-zaraz.md]:
```suggestion
If you need to block just one or more events from firing in a tool, we recommend you use [Blocking Triggers](/zaraz/get-started/create-trigger/#blocking-triggers).
```
You should only ensure that there's a slash right before the fragment (that is, before `#blocking-triggers`).

File: content/zaraz/reference/properties-reference.md [hunk 2]
```diff
@@ -20,50 +20,50 @@ Cloudflare Zaraz offers event and system properties that you can use when config
...
-| `{{ system.device.user-agent.device }}`          | String | Returns the type of device used (for example, iPhone).                                                         |
-| `{{ system.device.user-agent.cpu }}`             | String | Returns the device’s CPU.                                                                                      |
-| `{{ system.misc.random }}`                       | Number | Returns a random number unique to each request.                                                                |
-| `{{ system.misc.timestamp }}`                    | Number | Returns Unix time in seconds.                                                                                  |
-| `{{ system.misc.timestampMilliseconds }}`        | Number | Returns Unix time in milliseconds.                                                                             |
+| Property | Type | Description |
+| --- | --- | --- |
+| `{{ system.page.query.QUERY_PARAM }}` | String | Returns the value of a URL query parameter. Replace QUERY_PARAM with the name of your parameter. |
```
Comment (pedrosousa) [inline content/zaraz/reference/properties-reference.md]:
```suggestion
| `{{ system.page.query.QUERY_PARAM }}` | String | Returns the value of a URL query parameter. Replace `QUERY_PARAM` with the name of your parameter. |
```

File: content/zaraz/reference/properties-reference.md [hunk 3]
```diff
@@ -20,50 +20,50 @@ Cloudflare Zaraz offers event and system properties that you can use when config
...
 
 ## Event properties
 
 {{<table-wrap>}}
 
-| Property                    | Type   | Description                                                                                                                                                                                             |
-| --------------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Property | Type | Description |
+| --- | --- | --- |
 | `{{ client.__zarazTrack }}` | String | Returns the name of the event sent using the Track method of the Web API. Refer to [Zaraz Track](/zaraz/web-api/zaraz-track/) for more information.                                                     |
```
Comment (pedrosousa) [inline content/zaraz/reference/properties-reference.md]:
```suggestion
| `{{ client.__zarazTrack }}` | String | Returns the name of the event sent using the Track method of the Web API. Refer to [Zaraz Track](/zaraz/web-api/zaraz-track/) for more information. |
```

**Review by pedrosousa — APPROVED** [review]

File: content/zaraz/advanced/block-zaraz.md (hunk 1 above)
Comment (marciocloudflare) [reply content/zaraz/advanced/block-zaraz.md]: Got it :)

**Pull request merged**